contracts/cw721-non-transferable/src/msg.rs

```diff
@@ -0,0 +1,97 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Addr, Empty};
+use cw721_base::msg::QueryMsg as Cw721QueryMsg;
+
+#[cw_serde]
+pub struct InstantiateMsg {
+    pub admin: Option<String>,
+    pub name: String,
+    pub symbol: String,
+    pub minter: String,
+}
+
+#[cw_serde]
+pub enum QueryMsg {
+    GetAdmin {},
```

Comment:
I would just call this `Admin` instead of `GetAdmin`. All other query messages are noun phrases, while only this one is a verb phrase.